api-reference/10 UI Widgets/dxDiagram/1 Configuration/nodes/typeExpr.md

```diff
@@ -11,3 +11,21 @@ Specifies the name of a data source field or an expression that provides the sha
 The current node's data object.
 
 ---
+The built-in shape types are demonstrated in the [Shape Types](/concepts/05%20Widgets/Diagram/40%20Shape%20Types.md '/Documentation/Guide/Widgets/Diagram/Shape_Types/') section.
```

Comment:
Why "demonstrated"?

Reply:
Because the main purpose of the Shape Types topic is to demonstrate shape appearances (the topic contains an image for every shape type).

Reply:
```suggestion
The built-in shape types are shown in the [Shape Types](/concepts/05%20Widgets/Diagram/40%20Shape%20Types.md '/Documentation/Guide/Widgets/Diagram/Shape_Types/') section.
```